src/app/build.gradle

```diff
@@ -90,15 +90,15 @@ task compileFrontEnd() {
     }
     doLast {
         exec {
-            commandLine "npm", "run", "build", "--prefix=$projectDir/static/"
+            commandLine 'npm.cmd', "run", "build", "--prefix=$projectDir/static/"
```

Comment:
So this doesn't work on unix. Can we do something like this? (I'm not quite sure the exact syntax we'll need here)

```
String npm = 'npm';
if (Os.isFamily(Os.FAMILY_WINDOWS)) {
    npm = 'npm.cmd'
}
doLast {
    exec {
        commandLine npm, "run", "build", "--prefix=$projectDir/static/"
    }
}
```

Reply:
Added a check to determine npm command using the isWindows method on gradle's OperatingSystem class. I was able to run it locally and tests are now passing so that should cover both cases.